r/tests/testthat/test-dplyr-summarize.R

```diff
@@ -229,6 +229,60 @@ test_that("Group by n_distinct() on dataset", {
   )
 })
 
+test_that("median()", {
+  # with groups
+  expect_dplyr_equal(
+    input %>%
+      group_by(some_grouping) %>%
+      summarize(
+        med_dbl = median(dbl),
+        med_int = median(int),
+        med_dbl_narmf = median(dbl, FALSE),
+        # styler: off
+        med_int_narmf = median(int, na.rm = F),
```

Comment:
Why are you using F and T here?

Reply:
At an earlier stage of this work I was mucking around with the NSE in some even more exotic ways to try to get this all to work without ARROW-14050 or ARROW-14052. That introduced a sensitivity on the exact quoted value of `na.rm`. I subsequently removed that, so this isn't needed any more. I removed the `T` and `F` tests.